bedrock/firefox/templates/firefox/australis/base.html

```diff
@@ -53,3 +53,6 @@
 {% block site_footer %}{% endblock %}
 {% block email_form %}{% endblock %}
 {% block js %}{% endblock %}
+
+{# Bug 1381776 #}
+{% block update_notification %}{% endblock %}
```

Comment:
I would like to keep this on the radar, as implied opt-in feels risky, and explicit opt-out in templates can be difficult to manage and easy to forget. (I can see a quick-turnaround /new variation requested that forgets to/doesn't know to explicitly opt-out.)